doc/classes/CollisionShape2D.xml

```diff
@@ -22,6 +22,8 @@
 		</member>
 		<member name="one_way_collision" type="bool" setter="set_one_way_collision" getter="is_one_way_collision_enabled" default="false">
 			Sets whether this collision shape should only detect collision on one side (top or bottom).
+			If [code]true[/code], only edges that face up, relative to [CollisionShape2D]'s rotation, will collide with other objects.
+			When [code]true[/code], a red vector arrow will appear on the [CollisionShape2D] in the 2D screen.
```

Comment:
This line should be part of the above, no need for a separate paragraph (each line is a paragraph)

Reply:
Done.